README.md

```diff
@@ -14,4 +14,16 @@ For more details about using vizceral in your own projects with your own data, r
    npm run dev
    ```
 
-2. Open `localhost:8080` in your browser.
\ No newline at end of file
+2. Open `localhost:8080` in your browser.
+
+##### Using Docker
+If you don't have a node environment setup or would like to run this example on a platform, there is a Dockerfile for experimental usage.
+
+```
+$ docker build -t <name>/vizceral-example
```

Comment:
I needed to run `docker build -t <name>/vizceral-example .` (note the trailing dot).